tox.ini

```diff
@@ -93,7 +93,6 @@ deps=sphinx
      PyYAML
 
 commands=
-    {envpython} plugins_index/plugins_index.py
```

Comment:
@nicoddemus I suggested removing this when @hpk42 had trouble building the docs - I think it's okay to remove it as the plugin index got replaced by plugincompat on Heroku anyways, correct?